changelog.d/11678.bugfix

```diff
@@ -0,0 +1 @@
+Fix the documentation link in newly-generated configuration files.
```

Comment:
Should this be a `.doc` file?

Reply:
I was not sure. It is in code that generates something (among other things a broken link) that is essentially documentation. `.doc` was the other choice during my deliberation.

Reply:
From the point of view of someone reading our release notes I think this just fixes documentation, so let's go with `.doc`! 👍